samples/react/src/components/SiteLogo/index.js

```diff
@@ -3,10 +3,14 @@ import logo from '../../images/logo.svg';
 
 const SiteLogo = () => {
   return (
-    <a className="logo">
-      <img src={logo} alt="" style={{
-        width: '160px'
-      }} />
+    <a href="/" className="logo">
+      <img
+        src={logo}
+        alt=""
+        style={{
+          width: '160px',
+        }}
```

Comment:
[fix] I know this is outside the scope of this PR, but since it's being added to our template, it's worth fixing:

- Can you specify default alt text? Even something like "Company logo" would be better than nothing.
- Can you remove the inline style?

-

Reply:
I can do it.